packages/react-shortcuts/src/tests/ShortcutManager.test.tsx

```diff
@@ -120,13 +120,24 @@ describe('ShortcutManager', () => {
   it('calls shortcuts that are scoped to a specific node only when that node is focused', () => {
     const spy = jest.fn();
 
-    const app = mount(
+    const ShortcutWithFocusAndProvider = ({spy, focusNode}) => (
       <ShortcutProvider>
-        <ShortcutWithFocus spy={spy} />
-      </ShortcutProvider>,
+        <ShortcutWithFocus spy={spy} focusNode={focusNode} />
```

Comment:
We had a false positive here: 
- ref in `ShortcutWithFocus` was initialized with `null`
- `useShortcut` hook was called with `node: null` (not what we expected)
- `onMatch` spy was called because of the `partiallyMatching` code path (not `partiallyMatching && onFocusedNode` because `node` is null)